server/testutils/mockQuestionResponse.ts

```diff
@@ -46,4 +49,29 @@ const mockQuestionResponse = ({
   )
 }
 
-export default mockQuestionResponse
+export const mockOptionalQuestionResponse = ({
+  releaseType = 'other',
+  alternativeRadius,
+  duration,
+}: {
+  releaseType?: string
+  duration?: string
+  alternativeRadius?: string
+}) => {
+  ;(retrieveOptionalQuestionResponseFromApplicationOrAssessment as jest.Mock).mockImplementation(
+    // eslint-disable-next-line consistent-return
```

Comment:
What do you think about turning `consistent return` off globally?

Reply:
I think it _can_ be useful. If it becomes an annoyance we can think again.